apps/codeforafrica/src/components/NavList/NavListItem.js

```diff
@@ -0,0 +1,93 @@
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+const NavLinkGrid = styled("li")(({ theme: { breakpoints, typography } }) => ({
+  margin: `${typography.pxToRem(20)} 0`,
+  listStyle: "none",
+  [breakpoints.up("lg")]: {
+    margin: `${typography.pxToRem(10)} 0`,
+    listStyle: "none",
+  },
+}));
+
+const NavLink = styled("a", {
+  slot: "Root",
+  overridesResolver: (props, styles) => {
+    const { ownerState } = props;
+    return [styles.root, ownerState.footer && styles.footer];
+  },
+})(({ theme: { breakpoints, palette, typography }, ownerState }) => ({
+  textTransform: "capitalize",
+  color: "white",
+  textDecoration: "none",
+  fontSize: typography.pxToRem(28),
+  backgroundColor: "transparent",
+  "&:hover, &:active, &:focus, &:focus-within": {
+    color: "white",
+  },
+  [breakpoints.up("lg")]: {
+    textTransform: "capitalize",
+    color: "white",
+    textDecoration: "none",
+    fontSize: typography.pxToRem(23),
+    margin: typography.pxToRem(24),
+    fontweight: 700,
+    lineHeight: typography.pxToRem(28),
+    backgroundColor: "transparent",
+    borderColor: "none",
+    "&:hover, &:active, &:focus, &:focus-within": {
+      color: "white",
+    },
+  },
+  ...(!ownerState.footer && {
+    textTransform: "capitalize",
+    color: palette.text.secondary,
+    textDecoration: "none",
+    fontSize: typography.pxToRem(28),
+    backgroundColor: "transparent",
+    "&:hover, &:active, &:focus, &:focus-within": {
+      color: palette.text.secondary.main,
+    },
+    [breakpoints.up("lg")]: {
+      textTransform: "capitalize",
+      color: palette.secondary.main,
+      textDecoration: "none",
+      fontSize: typography.pxToRem(18),
+      margin: typography.pxToRem(24),
+      backgroundColor: "transparent",
+      borderColor: "none",
+      "&:hover, &:active, &:focus, &:focus-within": {
+        textDecoration: "underline",
+        color: palette.primary.main,
+      },
+    },
+  }),
+}));
+
+const NavListItem = React.forwardRef(function NavListItem(props, ref) {
```

Comment:
Since it'll be used in 2/3 places, I think it's best to have this as a standalone component in `src/components` folder.